OpenChange/MAPIStoreMailMessage.m

```diff
@@ -538,16 +541,45 @@ - (int) getPidTagNormalizedSubject: (void **) data
                           inMemCtx: (TALLOC_CTX *) memCtx
 {
   NSString *subject;
-  NSUInteger colIdx;
   NSString *stringValue;
 
   subject = [self subject];
-  colIdx = [subject rangeOfString: @":"].location;
-  if (colIdx != NSNotFound && colIdx < 4)
-    stringValue = [[subject substringFromIndex: colIdx + 1]
-                    stringByTrimmingLeadSpaces];
+  if (mailIsMeetingRequest)
+    {
+      NSUInteger quoteStartIdx, quoteEndIdx;
+      NSRange quoteRange;
```

Comment:
The declaration of variables within a if clause (frame) is not part of the coding style of this library. Although I like this style, uniform throughout the code wins in this case.